packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -350,11 +350,21 @@ class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>>
     }
   }
 
+  int _onChangedCallId = 0;
+  int _onChangedResultId = 0;
   // Called when _textEditingController changes.
   Future<void> _onChangedField() async {
+    final int callId = ++_onChangedCallId;
```

Comment:
`+=` is preferred, see https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#prefer--over-.

Reply:
Changes made